### PR TITLE
Edit the Species section in the View Project page

### PR DESCRIPTION
**Pull request by sdevalapurkar**
<details open>
  <summary><a href="https://quartech.atlassian.net/browse/BHBC-927" title="BHBC-927" target="_blank">BHBC-927</a></summary>
  <br />
  <table>
    <tr>
      <th>Summary</th>
      <td>Edit the Species section in the View Project page </td>
    </tr>
    <tr>
      <th>Type</th>
      <td>
        Story
        Story
      </td>
    </tr>
    <tr>
      <th>Status</th>
      <td>In Progress</td>
    </tr>
    <tr>
      <th>Points</th>
      <td>N/A</td>
    </tr>
    <tr>
      <th>Labels</th>
      <td>-</td>
    </tr>
  </table>
</details>

---

# Overview

## This PR contains the following changes

- Edit the Species section in the View Project page

## This PR contains the following types of changes

- [x] New feature (change which adds functionality)
- [x] Enhancement (improvements to existing functionality)

## How Has This Been Tested?

- Locally

### CODE REVIEW

File: api/src/queries/project/project-delete-queries.ts [hunk 1]
```diff
@@ -38,3 +38,75 @@ export const deleteIUCNSQL = (projectId: number): SQLStatement | null => {
...
+    return null;
+  }
+
+  const sqlStatement: SQLStatement = SQL`
+    DELETE
+      from focal_species
+    WHERE
+      p_id = ${projectId}
+    RETURNING
+      *;
```
Comment (NickPhura) [inline api/src/queries/project/project-delete-queries.ts]: Is this needed? What does it even return?

Comment (NickPhura) [reply api/src/queries/project/project-delete-queries.ts]: Will find out shortly as Im just adding the delete bit for project activities/climate

Comment (sdevalapurkar) [reply api/src/queries/project/project-delete-queries.ts]: yeah that can be removed now - I was trying some stuff but yea * returns all the rows deleted which doesn't really have to be returned rn

**Review by anissa-agahchen — APPROVED** [review]
Looks good

**Pull request merged**